src/components/WalletSelector/WalletButton.tsx

```diff
@@ -30,6 +30,26 @@ function WalletButton({
     activate(connector);
   }, [activate, connector, enableIsConnectingState, walletName]);
 
+  if (connector === injected) {
+    if (!(window.web3 || window.ethereum)) {
+      if (walletName.toLowerCase() === 'metamask') {
```

Comment:
would be helpful to see comments above each if statement explaining what they meant. as a first timer i'm not sure what `connector === injected` is...

Reply:
done ✅